### DISCUSSION TITLE
How to modify port 8080

### DISCUSSION THREAD

**Discussion by agy2009** [go-shiori/shiori#360] [Q&A] [answered]
Hello:

How to change port 8080 to other ports? Thank you

**Comment by kalifun** [comment, marked as answer]
Use the `-p` flag:

❯ go run main.go serve --help
Run a simple and performant web server which serves the site for managing bookmarks. If --port flag is not used, it will use port 8080 by default.

Usage:
  shiori serve [flags]

Flags:
  -a, --address string   Address the server listens to
  -h, --help             help for serve
  **-p, --port int         Port used by the server (default 8080)**
  -r, --webroot string   Root path that used by server (default "/")

Global Flags:
      --portable   run shiori in portable mode
